claimManagement/src/main/java/org/openimis/imisclaims/EnquireActivity.java

```diff
@@ -279,7 +289,8 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                         String CHFID = data.getStringExtra("SCAN_RESULT");
                         etCHFID.setText(CHFID);
 
-                        if (!CheckCHFID())return;
+                        Escape escape = new Escape();
+                        if (!escape.CheckCHFID())return;
```

Comment:
call escape.CheckCHFID(etCHFID.getText()) OR escape.CheckCHFID(tvCHFID.getText()) (don't know which one is here)